nrf_security/Kconfig

```diff
@@ -47,6 +47,31 @@ config MBEDTLS_TLS_LIBRARY
 	  Create the mbed SSL/TLS library in addition to the mbed crypto
 	  library.
 
+menu "mbed TLS memory configuration"
+
+config MBEDTLS_ENABLE_HEAP
+	bool "Enable global heap for mbed TLS"
+	help
+	  This option enables the mbedtls to use the heap. This setting must
+	  be global so that various applications and libraries in Zephyr do not
+	  try to do this themselves as there can be only one heap defined
+	  in mbedtls. If this is enabled, then the Zephyr will, during the device
+	  startup, initialize the heap automatically.
+
+config MBEDTLS_HEAP_SIZE
+	int "Heap size for mbed TLS"
```

Comment:
In bytes?

Reply:
no, echo_server uses `80000` bytes afaik